Core/src/org/sleuthkit/autopsy/core/UserPreferences.java

```diff
@@ -181,6 +183,14 @@ public static boolean displayTimesInLocalTime() {
     public static void setDisplayTimesInLocalTime(boolean value) {
         preferences.putBoolean(DISPLAY_TIMES_IN_LOCAL_TIME, value);
     }
+    
+    public static String getCustomTimeZone() {
+        return preferences.get(CUSTOM_TIME_ZONE, TimeZone.GMT_ZONE.getID());
+    }
+    
+    public static void setCustomTimeZone(String timeZone) {
+        preferences.put(CUSTOM_TIME_ZONE, timeZone);
+    }
```

Comment:
@dgrove727 and @wschaeferB, I am concerned that this name is not very descriptive. I find myself asking "Set custom time zone for what?" and  "What is a custom time zone, i.e., what makes it custom and if it is custom why is the default GMT?" Instead, it seems to me that it would be nice to try to make the implicit link between the DISPLAY_TIMES_IN_LOCAL_TIME and this setting more explicit. Perhaps something like set/getTimeZoneForDisplays. Then we would have code like

 if (!shouldDisplayTimesInLocalTime()) {
       return TimeZone.getTimeZone(UserPreferences.getTimeZoneForDisplays());
}